src/services/shopify-client.test.ts

```diff
@@ -0,0 +1,162 @@
+import { AxiosStatic } from 'axios';
+import { HttpShopifyClient, SubscriberListState } from './shopify-client';
+import { RefObject } from 'react';
+import { AppSession } from './app-session';
+import { DopplerLegacyUserData } from './doppler-legacy-client';
+
+function createHttpShopifyClient(axios: any) {
+  const axiosStatic = {
+    create: () => axios,
+  } as AxiosStatic;
+  const connectionDataRef = {
+    current: {
+      status: 'authenticated',
+      jwtToken: 'jwtToken',
+      userData: {} as DopplerLegacyUserData,
+    },
+  } as RefObject<AppSession>;
+  const baseUrl = 'http://shopify.test';
```

Comment:
maybe dont need and use in `new HttpShopifyClient({ axiosStatic, 'http://shopify.test', connectionDataRef })`

Reply:
Fixed